src/main/java/com/criteo/nosql/casspoke/cassandra/CassandraMonitor.java

```diff
@@ -71,6 +71,10 @@ private CassandraMonitor(final Service service, final Cluster cluster, final Whi
                 clusterBuilder.withAuthProvider(authProvider.get());
             }
 
+            if (service.getClusterName().contains("scylla")) {
```

Comment:
I would prefer to have an entry on the configuration to indicate if we want to enable tls or relying on a tag added to the scylla cluster to know if we should activate tls than relying on scylla cluster naming.
This feature can also benefits for our cassandra user also if we have scylla user relying on it it will broke it if they don't use tls

Reply:
Updated PR